math/casadi/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0   
+PortGroup           cmake 1.0
+
+github.setup        casadi casadi 3.3.0
+categories          math
+platforms           darwin
+license             LGPL-3.0
+maintainers         {@ierofant gmail.com:aw.kornilov} openmaintainer
+
+description         CasADi is a symbolic framework for numeric optimization
+long_description    CasADi is a symbolic framework for numeric optimization implementing automatic differentiation \
+                    in forward and reverse modes on sparse matrix-valued computational graphs. \
+                    It supports self-contained C-code generation and interfaces state-of-the-art codes \
+                    such as SUNDIALS, IPOPT etc. It can be used from C++, Python or Matlab/Octave.
+homepage            https://github.com/casadi/casadi/wiki
```

Comment:
The homepage should be covered by the GitHub portgroup, so this line should be removed.

Reply:
Done